cmd/src/setup.rs

```diff
@@ -208,3 +212,18 @@ pub fn validate_and_persist_config(config: &mut TiKvConfig, persist: bool) {
         }
     }
 }
+
+pub fn check_lock_file(config: &TiKvConfig) {
+    let file_path = Path::new(LOCK_FILE_BASE).join(format!("{}.tikv.lock", config.server.addr));
+
+    let fd = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .open(file_path)
+        .expect("open lock file failed")
+        .into_raw_fd();
+
+    if unsafe { libc::flock(fd, libc::LOCK_EX | libc::LOCK_NB) } != 0 {
```

Comment:
Does tikv need to support Windows and Mac, and if so do they need similar verification? 
 I will add the platform judgment here later

Reply:
You can copy the implement of "LOCK" file in tikv-server.